docs/rules/extensions.md

```diff
@@ -170,3 +170,10 @@ import foo from '@/foo';
 ## When Not To Use It
 
 If you are not concerned about a consistent usage of file extension.
+
+You should not configure this rule to be "never" if you are using ECMAScript modules directly in `node`.
```

Comment:
I don't think this is exactly accurate advice - when the node resolver does eventually support "exports" and ESM, "never" would not apply to anything that required the extension, but would correctly forbid it for anything where the extension was optional.

Reply:
The documentation for NodeJS has marked _Modules: ECMAScript modules_ as _stable_ since v15.3 - although other parts of the ESM spec are still marked experimental. They've explicitly required the `.js` extension for the ECMAScript loader [since v12](https://nodejs.org/docs/latest-v12.x/api/esm.html#esm_mandatory_file_extensions) (back in 2019).

Do you know that they're actually planning future changes to this part of the `import` behaviour specifically? Having something about the current behaviour in NodeJS would be useful!